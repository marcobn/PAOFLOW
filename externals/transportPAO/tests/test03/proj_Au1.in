&projwfc
  prefix='gold1'
  outdir='./SCRATCH'
  filpdos='./SCRATCH/gold1'
  lwrite_overlaps = .TRUE.
/
