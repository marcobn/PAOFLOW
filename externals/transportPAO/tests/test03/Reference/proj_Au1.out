
     Program PROJWFC v.5.0.2 (svn rev. 9392) starts on 17Apr2014 at 20:19:16 

     This program is part of the open-source Quantum ESPRESSO suite
     for quantum simulation of materials; please cite
         "P. Giannozzi et al., J. Phys.:Condens. Matter 21 395502 (2009);
          URL http://www.quantum-espresso.org", 
     in publications or presentations arising from this work. More details at
     http://www.quantum-espresso.org/quote.php

     Parallel version (MPI), running on     6 processors
     R & G space division:  proc/nbgrp/npool/nimage =       6

   Info: using nr1, nr2, nr3 values from input

   Info: using nr1s, nr2s, nr3s values from input

     IMPORTANT: XC functional enforced from input :
     Exchange-correlation      =  SLA  PW   GGX  GGC ( 1 4 2 2 0)
     EXX-fraction              =        0.00
     Any further DFT definition will be discarded
     Please, verify this is what you really want

 
     Parallelization info
     --------------------
     sticks:   dense  smooth     PW     G-vecs:    dense   smooth      PW
     Min        1061    1061    316                15056    15056    2465
     Max        1063    1063    317                15057    15057    2466
     Sum        6369    6369   1901                90341    90341   14791
 

     Check: negative/imaginary core charge=   -0.000021    0.000000

     Gaussian broadening (read from file): ngauss,degauss=   0    0.010000


     Calling projwave .... 

     Atomic states used for projection
     (read from pseudopotential files):

     state #   1: atom   1 (Au ), wfc  1 (l=0 m= 1)
     state #   2: atom   1 (Au ), wfc  2 (l=1 m= 1)
     state #   3: atom   1 (Au ), wfc  2 (l=1 m= 2)
     state #   4: atom   1 (Au ), wfc  2 (l=1 m= 3)
     state #   5: atom   1 (Au ), wfc  3 (l=2 m= 1)
     state #   6: atom   1 (Au ), wfc  3 (l=2 m= 2)
     state #   7: atom   1 (Au ), wfc  3 (l=2 m= 3)
     state #   8: atom   1 (Au ), wfc  3 (l=2 m= 4)
     state #   9: atom   1 (Au ), wfc  3 (l=2 m= 5)

 k =   0.0000000000  0.0000000000  0.0000000000
==== e(   1) =   -10.86449 eV ==== 
     psi = 0.762*[#   5]+0.234*[#   1]+
    |psi|^2 = 0.996
==== e(   2) =    -6.81588 eV ==== 
     psi = 1.000*[#   8]+
    |psi|^2 = 1.000
==== e(   3) =    -6.81578 eV ==== 
     psi = 1.000*[#   9]+
    |psi|^2 = 1.000
==== e(   4) =    -6.45860 eV ==== 
     psi = 0.759*[#   1]+0.237*[#   5]+
    |psi|^2 = 0.996
==== e(   5) =    -5.23889 eV ==== 
     psi = 0.921*[#   6]+0.079*[#   7]+
    |psi|^2 = 1.000
==== e(   6) =    -5.23889 eV ==== 
     psi = 0.921*[#   7]+0.079*[#   6]+
    |psi|^2 = 1.000
==== e(   7) =    -1.17759 eV ==== 
     psi = 0.723*[#   4]+0.203*[#   3]+
    |psi|^2 = 0.926
==== e(   8) =    -1.17759 eV ==== 
     psi = 0.723*[#   3]+0.203*[#   4]+
    |psi|^2 = 0.926
==== e(   9) =     1.14483 eV ==== 
     psi = 0.002*[#   1]+
    |psi|^2 = 0.002
==== e(  10) =     2.11447 eV ==== 
     psi = 
    |psi|^2 = 0.000

 k =   0.0000000000  0.0000000000  0.5291766857
==== e(   1) =   -10.06206 eV ==== 
     psi = 0.769*[#   5]+0.190*[#   1]+0.040*[#   2]+
    |psi|^2 = 0.999
==== e(   2) =    -6.72210 eV ==== 
     psi = 1.000*[#   8]+
    |psi|^2 = 1.000
==== e(   3) =    -6.72198 eV ==== 
     psi = 1.000*[#   9]+
    |psi|^2 = 1.000
==== e(   4) =    -6.22622 eV ==== 
     psi = 0.581*[#   1]+0.229*[#   5]+0.189*[#   2]+
    |psi|^2 = 0.999
==== e(   5) =    -5.94873 eV ==== 
     psi = 0.911*[#   6]+0.062*[#   7]+0.023*[#   3]+0.002*[#   4]+
    |psi|^2 = 0.997
==== e(   6) =    -5.94873 eV ==== 
     psi = 0.911*[#   7]+0.062*[#   6]+0.023*[#   4]+0.002*[#   3]+
    |psi|^2 = 0.997
==== e(   7) =    -0.47850 eV ==== 
     psi = 0.742*[#   3]+0.176*[#   4]+0.021*[#   6]+0.005*[#   7]+
    |psi|^2 = 0.945
==== e(   8) =    -0.47850 eV ==== 
     psi = 0.742*[#   4]+0.176*[#   3]+0.021*[#   7]+0.005*[#   6]+
    |psi|^2 = 0.945
==== e(   9) =     1.51296 eV ==== 
     psi = 0.234*[#   2]+0.065*[#   1]+
    |psi|^2 = 0.299
==== e(  10) =     2.48309 eV ==== 
     psi = 
    |psi|^2 = 0.000

 k =   0.0000000000  0.0000000000  1.0583533715
==== e(   1) =    -7.87725 eV ==== 
     psi = 0.862*[#   5]+0.073*[#   1]+0.062*[#   2]+
    |psi|^2 = 0.997
==== e(   2) =    -7.20890 eV ==== 
     psi = 0.691*[#   6]+0.287*[#   7]+0.014*[#   3]+0.006*[#   4]+
    |psi|^2 = 0.998
==== e(   3) =    -7.20890 eV ==== 
     psi = 0.691*[#   7]+0.287*[#   6]+0.014*[#   4]+0.006*[#   3]+
    |psi|^2 = 0.998
==== e(   4) =    -6.48888 eV ==== 
     psi = 1.000*[#   8]+
    |psi|^2 = 1.000
==== e(   5) =    -6.48873 eV ==== 
     psi = 1.000*[#   9]+
    |psi|^2 = 1.000
==== e(   6) =    -5.49805 eV ==== 
     psi = 0.506*[#   1]+0.352*[#   2]+0.137*[#   5]+
    |psi|^2 = 0.995
==== e(   7) =     1.04006 eV ==== 
     psi = 0.916*[#   3]+0.029*[#   4]+0.020*[#   6]+
    |psi|^2 = 0.965
==== e(   8) =     1.04006 eV ==== 
     psi = 0.916*[#   4]+0.029*[#   3]+0.020*[#   7]+
    |psi|^2 = 0.965
==== e(   9) =     2.61879 eV ==== 
     psi = 0.178*[#   2]+0.101*[#   1]+
    |psi|^2 = 0.279
==== e(  10) =     3.58891 eV ==== 
     psi = 
    |psi|^2 = 0.000

 k =   0.0000000000  0.0000000000  1.5875300572
==== e(   1) =    -8.18047 eV ==== 
     psi = 0.773*[#   7]+0.222*[#   6]+0.004*[#   4]+0.001*[#   3]+
    |psi|^2 = 0.999
==== e(   2) =    -8.18047 eV ==== 
     psi = 0.773*[#   6]+0.222*[#   7]+0.004*[#   3]+0.001*[#   4]+
    |psi|^2 = 0.999
==== e(   3) =    -6.24526 eV ==== 
     psi = 1.000*[#   8]+
    |psi|^2 = 1.000
==== e(   4) =    -6.24508 eV ==== 
     psi = 1.000*[#   9]+
    |psi|^2 = 1.000
==== e(   5) =    -5.83945 eV ==== 
     psi = 0.950*[#   5]+0.049*[#   1]+
    |psi|^2 = 0.999
==== e(   6) =    -3.36722 eV ==== 
     psi = 0.557*[#   2]+0.396*[#   1]+0.021*[#   5]+
    |psi|^2 = 0.974
==== e(   7) =     3.03271 eV ==== 
     psi = 0.581*[#   4]+0.334*[#   3]+0.003*[#   7]+0.001*[#   6]+
    |psi|^2 = 0.919
==== e(   8) =     3.03271 eV ==== 
     psi = 0.581*[#   3]+0.334*[#   4]+0.003*[#   6]+0.001*[#   7]+
    |psi|^2 = 0.919
==== e(   9) =     4.47344 eV ==== 
     psi = 0.081*[#   2]+0.030*[#   1]+
    |psi|^2 = 0.112
==== e(  10) =     5.43156 eV ==== 
     psi = 
    |psi|^2 = 0.000

 k =   0.0000000000  0.0000000000 -2.1167067430
==== e(   1) =    -8.53427 eV ==== 
     psi = 0.911*[#   6]+0.088*[#   7]+
    |psi|^2 = 0.999
==== e(   2) =    -8.53427 eV ==== 
     psi = 0.911*[#   7]+0.088*[#   6]+
    |psi|^2 = 0.999
==== e(   3) =    -6.14105 eV ==== 
     psi = 1.000*[#   8]+
    |psi|^2 = 1.000
==== e(   4) =    -6.14088 eV ==== 
     psi = 1.000*[#   9]+
    |psi|^2 = 1.000
==== e(   5) =    -5.35446 eV ==== 
     psi = 0.873*[#   5]+0.127*[#   1]+
    |psi|^2 = 1.000
==== e(   6) =    -0.27653 eV ==== 
     psi = 0.968*[#   2]+
    |psi|^2 = 0.968
==== e(   7) =     2.06667 eV ==== 
     psi = 0.838*[#   1]+0.121*[#   5]+
    |psi|^2 = 0.959
==== e(   8) =     4.75265 eV ==== 
     psi = 0.734*[#   3]+0.104*[#   4]+
    |psi|^2 = 0.838
==== e(   9) =     4.75265 eV ==== 
     psi = 0.734*[#   4]+0.104*[#   3]+
    |psi|^2 = 0.838
==== e(  10) =     6.99971 eV ==== 
     psi = 
    |psi|^2 = 0.001

 k =   0.0000000000  0.0000000000 -1.5875300572
==== e(   1) =    -8.18047 eV ==== 
     psi = 0.992*[#   6]+0.005*[#   3]+0.003*[#   7]+
    |psi|^2 = 0.999
==== e(   2) =    -8.18047 eV ==== 
     psi = 0.992*[#   7]+0.005*[#   4]+0.003*[#   6]+
    |psi|^2 = 0.999
==== e(   3) =    -6.24526 eV ==== 
     psi = 1.000*[#   8]+
    |psi|^2 = 1.000
==== e(   4) =    -6.24508 eV ==== 
     psi = 1.000*[#   9]+
    |psi|^2 = 1.000
==== e(   5) =    -5.83945 eV ==== 
     psi = 0.950*[#   5]+0.049*[#   1]+
    |psi|^2 = 0.999
==== e(   6) =    -3.36722 eV ==== 
     psi = 0.557*[#   2]+0.396*[#   1]+0.021*[#   5]+
    |psi|^2 = 0.974
==== e(   7) =     3.03271 eV ==== 
     psi = 0.864*[#   3]+0.051*[#   4]+0.004*[#   6]+
    |psi|^2 = 0.919
==== e(   8) =     3.03271 eV ==== 
     psi = 0.864*[#   4]+0.051*[#   3]+0.004*[#   7]+
    |psi|^2 = 0.919
==== e(   9) =     4.47344 eV ==== 
     psi = 0.081*[#   2]+0.030*[#   1]+
    |psi|^2 = 0.112
==== e(  10) =     5.43156 eV ==== 
     psi = 
    |psi|^2 = 0.000

 k =   0.0000000000  0.0000000000 -1.0583533715
==== e(   1) =    -7.87725 eV ==== 
     psi = 0.862*[#   5]+0.073*[#   1]+0.062*[#   2]+
    |psi|^2 = 0.997
==== e(   2) =    -7.20890 eV ==== 
     psi = 0.978*[#   6]+0.020*[#   3]+
    |psi|^2 = 0.998
==== e(   3) =    -7.20890 eV ==== 
     psi = 0.978*[#   7]+0.020*[#   4]+
    |psi|^2 = 0.998
==== e(   4) =    -6.48888 eV ==== 
     psi = 1.000*[#   8]+
    |psi|^2 = 1.000
==== e(   5) =    -6.48873 eV ==== 
     psi = 1.000*[#   9]+
    |psi|^2 = 1.000
==== e(   6) =    -5.49805 eV ==== 
     psi = 0.506*[#   1]+0.352*[#   2]+0.137*[#   5]+
    |psi|^2 = 0.995
==== e(   7) =     1.04006 eV ==== 
     psi = 0.873*[#   4]+0.072*[#   3]+0.019*[#   7]+0.002*[#   6]+
    |psi|^2 = 0.965
==== e(   8) =     1.04006 eV ==== 
     psi = 0.873*[#   3]+0.072*[#   4]+0.019*[#   6]+0.002*[#   7]+
    |psi|^2 = 0.965
==== e(   9) =     2.61879 eV ==== 
     psi = 0.178*[#   2]+0.101*[#   1]+
    |psi|^2 = 0.279
==== e(  10) =     3.58891 eV ==== 
     psi = 
    |psi|^2 = 0.000

 k =   0.0000000000  0.0000000000 -0.5291766857
==== e(   1) =   -10.06206 eV ==== 
     psi = 0.769*[#   5]+0.190*[#   1]+0.040*[#   2]+
    |psi|^2 = 0.999
==== e(   2) =    -6.72210 eV ==== 
     psi = 1.000*[#   8]+
    |psi|^2 = 1.000
==== e(   3) =    -6.72198 eV ==== 
     psi = 1.000*[#   9]+
    |psi|^2 = 1.000
==== e(   4) =    -6.22622 eV ==== 
     psi = 0.581*[#   1]+0.229*[#   5]+0.189*[#   2]+
    |psi|^2 = 0.999
==== e(   5) =    -5.94873 eV ==== 
     psi = 0.506*[#   7]+0.466*[#   6]+0.013*[#   4]+0.012*[#   3]+
    |psi|^2 = 0.997
==== e(   6) =    -5.94873 eV ==== 
     psi = 0.506*[#   6]+0.466*[#   7]+0.013*[#   3]+0.012*[#   4]+
    |psi|^2 = 0.997
==== e(   7) =    -0.47850 eV ==== 
     psi = 0.636*[#   4]+0.282*[#   3]+0.018*[#   7]+0.008*[#   6]+
    |psi|^2 = 0.945
==== e(   8) =    -0.47850 eV ==== 
     psi = 0.636*[#   3]+0.282*[#   4]+0.018*[#   6]+0.008*[#   7]+
    |psi|^2 = 0.945
==== e(   9) =     1.51296 eV ==== 
     psi = 0.234*[#   2]+0.065*[#   1]+
    |psi|^2 = 0.299
==== e(  10) =     2.48309 eV ==== 
     psi = 
    |psi|^2 = 0.000

Lowdin Charges: 

     Atom #   1: total charge =  10.9858, s =  0.9758, p =  0.3710, d =  9.6390, 
     Spilling Parameter:   0.0013
